### PR TITLE
locale fr_FR

### PR DESCRIPTION
**Pull request by forgive38**
We try to have a fr_FR version of OJS
So we started from fr_CA and added some littles corrections
Maybe it can be useful for other people
Please feel free of use it.

Thank you

Mathdoc

### CODE REVIEW

File: locale/fr_FR/author.xml [hunk 1]
```diff
@@ -21,22 +21,22 @@
 	<message key="author.submit.startHereTitle">Commencer une nouvelle soumission</message>
 	<message key="author.submit.startHereLink"><![CDATA[<a href="{$submitUrl}" class="action">Cliquer ici</a> pour vous rendre à la première étape du processus de soumission en cinq étapes.]]></message>
 	<message key="author.submit.step1">Étape 1. Commencer la soumission</message>
-	<message key="author.submit.step2">Étape 2. Télécharger la soumission</message>
+	<message key="author.submit.step2">Étape 2. Tranférer la soumission</message>
 	<message key="author.submit.step3">Étape 3. Saisir les métadonnées de la soumission</message>
-	<message key="author.submit.step4">Étape 4. Télécharger les fichiers supplémentaires</message>
+	<message key="author.submit.step4">Étape 4. Tranférer les fichiers supplémentaires</message>
```
Comment (mtub) [inline locale/fr_FR/author.xml]: Transférer?

File: locale/fr_FR/author.xml [hunk 2]
```diff
@@ -21,22 +21,22 @@
 	<message key="author.submit.startHereTitle">Commencer une nouvelle soumission</message>
 	<message key="author.submit.startHereLink"><![CDATA[<a href="{$submitUrl}" class="action">Cliquer ici</a> pour vous rendre à la première étape du processus de soumission en cinq étapes.]]></message>
 	<message key="author.submit.step1">Étape 1. Commencer la soumission</message>
-	<message key="author.submit.step2">Étape 2. Télécharger la soumission</message>
+	<message key="author.submit.step2">Étape 2. Tranférer la soumission</message>
```
Comment (mtub) [inline locale/fr_FR/author.xml]: Transférer?

File: locale/fr_FR/author.xml [hunk 3]
```diff
@@ -66,21 +66,21 @@
...
-	<message key="author.submit.supplementaryFiles.saveToUpload">Cliquer sur Enregistrer pour télécharger le fichier (après cela, des fichiers supplémentaires peuvent être téléchargés).</message>
-	<message key="author.submit.forgottenSubmitSuppFile">Êtes-vous certain de vouloir continuer sans télécharger le fichier supplémentaire que vous avez choisi?</message>
+	<message key="author.submit.supplementaryFiles.saveToUpload">Cliquer sur Enregistrer pour transférer le fichier (après cela, des fichiers supplémentaires peuvent être transférés).</message>
+	<message key="author.submit.forgottenSubmitSuppFile">Êtes-vous certain de vouloir continuer sans tranférer le fichier supplémentaire que vous avez choisi?</message>
```
Comment (mtub) [inline locale/fr_FR/author.xml]: Transférer?

File: locale/fr_FR/author.xml (hunk 2 above)
Comment (forgive38) [reply locale/fr_FR/author.xml]: "Transférer" is a french translation for Upload (better than "Télécharger" which means more "Download")

**Comment by forgive38** [comment]
"Transférer" is a french translation for Upload (better than "Télécharger" which means more "Download")

**Pull request merged**